catroid/src/main/java/org/catrobat/catroid/sensing/CollisionInformation.java

```diff
@@ -370,7 +370,7 @@ private static float pointToLineDistance(PointF lineStart, PointF lineEnd, Point
 	}
 
 	private static float pointToPointDistance(PointF p1, PointF p2) {
-		return (float) Math.sqrt(((p1.x - p2.x) * (p1.x - p2.x) + (p1.y - p2.y) * (p1.y - p2.y)));
+		return (float) Math.hypot(p1.x - p2.x, p1.y - p2.y);
```

Comment:
SUGGESTION: Consider using https://refactoring.guru/inline-method here instead. The code is so simple that it seems unnecessary to have a separate function here, which is called only once. Also, in the other case a unit test for the function would be needed --- an overkill here IMHO.

Reply:
I used the inline-method and removed the "pointToPointDistance" function, because it's not necessary now.
Thanks for this suggestion. and for the refrence ^_^